sites/docs/src/content/docs/usage/configuration/system_requirements.md

```diff
@@ -0,0 +1,121 @@
+---
+title: System requirements
```

Comment:
I think this title is confusing, system requirements often corresponds to what the pipeline needs to run, but rather this is 'Configuring for your system'

Reply:
Fair - will update